src/drivers/boards/px4fmu-v5/px4fmu_spi.c

```diff
@@ -214,9 +214,10 @@ __EXPORT int stm32_spi_bus_initialize(void)
 	/* Default PX4_SPI_BUS_MEMORY to 12MHz and de-assert the known chip selects.
 	 */
 
-	SPI_SETFREQUENCY(spi_memory, 12 * 1000 * 1000);
+	SPI_SETFREQUENCY(spi_memory, 20 * 1000 * 1000);
```

Comment:
The Ramtron driver  Starts off with speed = 0 That selects the lowest bus speed the CPU can do on spi then it reads the chip ID and switches to the max the chip can handle.